src/terraform/structure/terraform_parser.go

```diff
@@ -799,7 +815,38 @@ func getUncloseBracketsCount(bracketsCounters map[hclsyntax.TokenType]int) int {
 	return sum
 }
 
-func (p *TerraformParser) parseTagAttribute(tokens hclwrite.Tokens) map[string]string {
+func (p *TerraformParser) parseTagAttribute(tokens hclwrite.Tokens) (map[string]string, error) {
```

Comment:
Why change this func signature to return 2 args?

Reply:
@ChanochShayner 
the unmarshal function can return error so i think that  need to add option to return error
but if you think that it unnecessary i can remove it:
do you want to replace this:
 err := dethcl.Unmarshal([]byte(hclBytes), hclData)
to this: 
dethcl.Unmarshal([]byte(hclBytes), hclData)

Reply:
@ChanochShayner 
fix